packages/client/subscriptions/TaskSubscription.ts

```diff
@@ -32,56 +32,40 @@ const subscription = graphql`
 
 const onNextHandlers = {
   UpdateTaskPayload: updateTaskTaskOnNext
-}
+} as const
+
+const updateHandlers = {
+  ChangeTaskTeamPayload: changeTaskTeamTaskUpdater,
+  CreateTaskPayload: createTaskTaskUpdater,
+  DeleteTaskPayload: deleteTaskTaskUpdater,
+  EditTaskPayload: editTaskTaskUpdater,
+  RemoveOrgUserPayload: removeOrgUserTaskUpdater,
+  UpdateTaskPayload: updateTaskTaskUpdater
+} as const
 
 const TaskSubscription = (
   atmosphere: Atmosphere,
   variables: TaskSubscriptionVariables,
   router: {history: RouterProps['history']}
 ) => {
-  const {viewerId} = atmosphere
   return requestSubscription<TTaskSubscription>(atmosphere, {
     subscription,
     variables,
     updater: (store) => {
       const payload = store.getRootField('taskSubscription') as any
       if (!payload) return
-      const type = payload.getValue('__typename')
+      const type = payload.getValue('__typename') as keyof typeof updateHandlers
       const context = {atmosphere, store: store as RecordSourceSelectorProxy<any>}
-      switch (type) {
```

Comment:
now that all the updaters have the same signature `(payload, context)` we can use a lookup table above